modules/main.py

```diff
@@ -1,10 +1,12 @@
 import sys
 from threading import Thread
 
+from modules.battle import BattleHandler, check_lead_can_battle, RotatePokemon
 from modules.config import config, load_config_from_directory
 from modules.console import console
 from modules.context import context
 from modules.memory import get_game_state, GameState
+from modules.menuing import MenuWrapper, CheckForPickup, should_check_for_pickup
 from modules.pokemon import opponent_changed, get_opponent
 from modules.temp import temp_run_from_battle
```

Comment:
We don't need to have the tem_run_from_battle import, right?

Reply:
I'm testing on emerald. So far so good 👍

Reply:
Hey thanks for catching this! I've gone ahead and removed that import because you are correct- `temp_run_from_battle` no longer gets called in this file. Thanks for pointing this out!